aemedge/blocks/link-list/link-list.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Link List Block
+ */
+
+/**
+ * Wrap the text node of a link with a `span` element.
+ * (!) Text node can also be positioned before the icon.
+ * @param link {Element} The link element
+ */
+function moveLinkTextNodeToSpan(link) {
+  Array.from(link.childNodes)
+    .forEach((node) => {
+      if (node.nodeType === Node.TEXT_NODE && node.textContent.trim().length > 0) {
+        const span = document.createElement('span');
```

Comment:
use dom-builder.js. Also use decorateIcons for wrapping

Reply:
Used the dom-builder for span. Thanks. The wrapper itself is needed because the icon can also be placed on the left and right side of the text (need to control the margin).